src/AuthenticationService.php

```diff
@@ -227,7 +227,7 @@ public function clearIdentity(ServerRequestInterface $request, ResponseInterface
     {
         foreach ($this->_authenticators as $authenticator) {
             if ($authenticator instanceof PersistenceInterface) {
-                $authenticator->clearIdentity($request);
+                $authenticator->clearIdentity($request, $response);
```

Comment:
You should assign the response back, since any changes done to it would be lost otherwise